content/en/glossary/terms/approval_wait_time.md

```diff
@@ -5,4 +5,4 @@ core_product:
 related_terms:
   - queue time
 ---
-Approval wait time is the duration that workflow runs and jobs within a workflow are paused, pending manual approval. For more information, <a href="/continuous_integration/pipelines/#supported-features">see the documentation</a>.
\ No newline at end of file
+Approval wait time is the duration that pipelines and jobs or stages within a pipeline are blocked, pending manual approval. For more information, <a href="/continuous_integration/pipelines/#supported-features">see the documentation</a>.
```

Comment:
Updated the glossary as `workflow runs` is the term Github uses for pipelines. Since the glossary is a generic definition maybe it is better to use Datadog terms which are more generic. For us, the different CI-CD levels are, from bigger to smaller scope: pipeline -> stage -> job -> step.